news_app.js

```diff
@@ -0,0 +1,105 @@
+// const { doc } = require("prettier");
+
+// FETCH VARIABLES
+const topic = 'food';
+const source = `https://newsapi.org/v2/everything?q=${topic}&sortBy=popularity&pageSize=5&apiKey=b10b22f57fca436ca78791aa2c90a376`;
```

Comment:
save api key in different  file  file, you can read about .env files, this is dot-file and it is quite useful concept https://www.freecodecamp.org/news/dotfiles-what-is-a-dot-file-and-how-to-create-it-in-mac-and-linux/